tests/modules/programs/firefox/profiles/userchrome/default.nix

```diff
@@ -21,15 +18,39 @@ in
         basic.isDefault = true;
         lines = {
           id = 1;
-          userChrome = builtins.readFile ./chrome/userChrome.css;
+          userChrome = dummyUserChromeText;
         };
-        file = {
+        file-text = {
           id = 2;
-          userChrome = ./chrome/userChrome.css;
+          userChrome.text = dummyUserChromeText;
+        };
```

Comment:
It would be nice to see a backwards compatible test case of passing the file, still.

Reply:
What name do you think is missing?

Reply:
I meant, this no longer has the `file` test which had the path being passed in. There are 2 tests for the lines which are read from the file itself, one with `userChrome` and one with `userChrome.text`. `userChrome = file.css` throws an error, for me because it tries to read the contents of the file always.

Reply:
The `file` text has been replaced by `source-file` because these changes are additive to cba2f9ce95c8d10b66cacf05a275e3ad71959638 only.